hack/run-tests-in-container.sh

```diff
@@ -51,10 +51,14 @@ $KUBECTL_BINARY create clusterrolebinding functest-cluster-admin \
 
 $KUBECTL_BINARY -n "${INSTALLED_NAMESPACE}" delete pod functest --ignore-not-found --wait=true
 
+if [[ -n ${ARTIFACT_DIR} ]]; then
+  JUNIT_PARAM="--junit-output=${ARTIFACT_DIR}/junit/report.xml"
+fi
+
 $KUBECTL_BINARY -n "${INSTALLED_NAMESPACE}" run functest \
  --image="$FUNC_TEST_IMAGE" --serviceaccount=functest \
  --env="INSTALLED_NAMESPACE=${INSTALLED_NAMESPACE}" \
- --restart=Never -- --config-file hack/testFiles/test_config.yaml
+ --restart=Never -- --config-file hack/testFiles/test_config.yaml "${JUNIT_PARAM}"
```

Comment:
This path will be in the container. Who is going to fetch them from the container?

Reply:
got you.